### ISSUE TITLE
The timetable cannot be scrolled for a while after returning from the session details.

### ISSUE THREAD

**Issue by Corvus400** [DroidKaigi/conference-app-2024#859]
#543 

When I was dealing with the PR above, I could scroll instantly when I came back from details.
However, the current code somehow prevents scrolling until the animation is over.
I think it would be better to revert to the previous code. 🤔 

<video src="https://github.com/user-attachments/assets/5091070a-d4a2-417f-8117-d2c5c1fa8c43" width="300">

**Comment by takahirom** [comment]
Certainly, it might be more important for users to be able to interact with the application than to worry about minor display issues.
